<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1335</width>
    <height>806</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1330</width>
    <height>806</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>芯视界USB设备测试程序</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon_1.png</normaloff>icon_1.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame_4">
      <property name="styleSheet">
       <string notr="true">QFrame#frame_4{
border:1px solid grey;}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QFrame" name="frame">
           <property name="maximumSize">
            <size>
             <width>400</width>
             <height>523</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QGroupBox" name="groupBox">
                <property name="styleSheet">
                 <string notr="true">font:10pt &quot;黑体&quot;;</string>
                </property>
                <property name="title">
                 <string>TOF图像：</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_3">
                 <item row="0" column="0">
                  <widget class="QLabel" name="showTOF_label">
                   <property name="minimumSize">
                    <size>
                     <width>367</width>
                     <height>219</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">border:1px solid black;
background-color: rgb(0, 0, 0);</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_2">
                <property name="styleSheet">
                 <string notr="true">font:10pt &quot;黑体&quot;;</string>
                </property>
                <property name="title">
                 <string>Peak图像：</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_4">
                 <item row="0" column="0">
                  <widget class="QLabel" name="showIntensity_label">
                   <property name="minimumSize">
                    <size>
                     <width>367</width>
                     <height>219</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>358</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">border:1px solid black;
background-color: rgb(0, 0, 0);</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_3">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QGroupBox" name="groupBox_3">
              <property name="styleSheet">
               <string notr="true">font:10pt &quot;黑体&quot;;</string>
              </property>
              <property name="title">
               <string>点云图像：</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_6">
               <property name="leftMargin">
                <number>9</number>
               </property>
               <property name="topMargin">
                <number>9</number>
               </property>
               <item row="0" column="0">
                <widget class="GLWidget" name="widget" native="true">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>485</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">border:1px solid black;
background-color: rgb(0, 0, 0);</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_2">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>500</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <layout class="QGridLayout" name="gridLayout_12">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QGroupBox" name="groupBox_4">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>107</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">font:10pt &quot;黑体&quot;;</string>
                </property>
                <property name="title">
                 <string>设备链接：</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_8">
                 <item row="0" column="0">
                  <layout class="QHBoxLayout" name="horizontalLayout_2">
                   <item>
                    <widget class="QLabel" name="label">
                     <property name="text">
                      <string>VID:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="VID_lineEdit">
                     <property name="text">
                      <string>8888</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item row="1" column="0">
                  <layout class="QHBoxLayout" name="horizontalLayout_3">
                   <item>
                    <widget class="QLabel" name="label_2">
                     <property name="text">
                      <string>PID </string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="PID_lineEdit">
                     <property name="text">
                      <string>0160</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item row="2" column="0">
                  <layout class="QHBoxLayout" name="horizontalLayout_7">
                   <item>
                    <spacer name="horizontalSpacer">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QPushButton" name="pushButton">
                     <property name="styleSheet">
                      <string notr="true">font:10pt &quot;黑体&quot;;</string>
                     </property>
                     <property name="text">
                      <string>连接设备</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_6">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>106</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">font:10pt &quot;黑体&quot;;
</string>
                </property>
                <property name="title">
                 <string>系统寄存器：</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_9">
                 <item row="0" column="0">
                  <layout class="QHBoxLayout" name="horizontalLayout_4">
                   <item>
                    <widget class="QLabel" name="label_3">
                     <property name="text">
                      <string>地址：</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="lineEdit">
                     <property name="text">
                      <string>13</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item row="1" column="0">
                  <layout class="QHBoxLayout" name="horizontalLayout_5">
                   <item>
                    <widget class="QLabel" name="label_4">
                     <property name="text">
                      <string>数据：</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="lineEdit_2"/>
                   </item>
                  </layout>
                 </item>
                 <item row="2" column="0">
                  <layout class="QHBoxLayout" name="horizontalLayout_6">
                   <item>
                    <widget class="QPushButton" name="readSys_pushButton">
                     <property name="text">
                      <string>读取</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="writeSys_pushButton">
                     <property name="text">
                      <string>写入</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_7">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">font:10pt &quot;黑体&quot;;</string>
                </property>
                <property name="title">
                 <string>设备寄存器：</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_10">
                 <item row="0" column="0">
                  <layout class="QHBoxLayout" name="horizontalLayout_8">
                   <item>
                    <widget class="QLabel" name="label_5">
                     <property name="text">
                      <string>设备地址：</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="lineEdit_3">
                     <property name="text">
                      <string>D8</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item row="1" column="0">
                  <layout class="QHBoxLayout" name="horizontalLayout_9">
                   <item>
                    <widget class="QLabel" name="label_7">
                     <property name="text">
                      <string>    地址：</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="lineEdit_4"/>
                   </item>
                  </layout>
                 </item>
                 <item row="2" column="0">
                  <layout class="QHBoxLayout" name="horizontalLayout_10">
                   <item>
                    <widget class="QLabel" name="label_8">
                     <property name="text">
                      <string>    数据：</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="lineEdit_5"/>
                   </item>
                  </layout>
                 </item>
                 <item row="3" column="0">
                  <layout class="QHBoxLayout" name="horizontalLayout_11">
                   <item>
                    <widget class="QPushButton" name="readDev_pushButton">
                     <property name="minimumSize">
                      <size>
                       <width>0</width>
                       <height>20</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>读取</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="writeDev_pushButton">
                     <property name="minimumSize">
                      <size>
                       <width>0</width>
                       <height>20</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>写入</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_8">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>82</height>
                 </size>
                </property>
                <property name="title">
                 <string>配置集：</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_11">
                 <item row="0" column="0">
                  <layout class="QVBoxLayout" name="verticalLayout_2">
                   <item>
                    <widget class="QPushButton" name="loadSetting_pushButton">
                     <property name="text">
                      <string>加载配置集</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="saveSetting_pushButton">
                     <property name="text">
                      <string>保存配置集</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_12">
                <item>
                 <widget class="QPushButton" name="pushButton_2">
                  <property name="styleSheet">
                   <string notr="true">font:10pt &quot;黑体&quot;;</string>
                  </property>
                  <property name="text">
                   <string>播放</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_5">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>180</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font:10pt &quot;黑体&quot;;</string>
      </property>
      <property name="title">
       <string>监控窗口：</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_16">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QTabWidget" name="tabWidget">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="tab">
            <attribute name="title">
             <string>连接信息</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_13">
             <item row="0" column="1">
              <widget class="QTextEdit" name="textEdit"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_2">
            <attribute name="title">
             <string>运行日志</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_17">
             <item row="0" column="0">
              <widget class="QTextEdit" name="textEdit_2"/>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_6">
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_15">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>           TOF/PEAK信息统计</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTableWidget" name="tableWidget_2">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <row>
                 <property name="text">
                  <string>tof</string>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string>peak</string>
                 </property>
                </row>
                <column>
                 <property name="text">
                  <string>minValue</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>maxValue</string>
                 </property>
                </column>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_5">
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_14">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>             空间信息统计</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTableWidget" name="tableWidget_4">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <row>
                 <property name="text">
                  <string>x</string>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string>y</string>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string>z</string>
                 </property>
                </row>
                <column>
                 <property name="text">
                  <string>minValue</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>maxValue</string>
                 </property>
                </column>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1335</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>相机属性</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>设置</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header location="global">glwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
